src/api/subscriptions.js

```diff
@@ -362,17 +370,24 @@ export const PublisherEntitlementEvent = {
 };
 
 /**
- * PublisherEntitlement
- *   In order to participate in News Showcase, publishers must report information about their entitlements.
- * Properties:
- * - isUserRegistered: Is the user registered currently?
- * - entitlement: Publisher entitlement event type.
- *  @typedef {{
- *    isUserRegistered:  !boolean,
- *    entitlement: !PublisherEntitlementEvent,
+ * Showcase publishers log these events with optional metadata.
+ * @typedef {{
+ *   event: !ShowcaseEvent,
+ *   isUserRegistered: (boolean|undefined),
+ *   isFromUserAction: (boolean|undefined),
+ * }}
+ */
+export let ShowcaseEventWithMetadata;
+
+/**
+ * DEPRECATED
+ * Showcase publishers log these events with optional metadata.
+ * @typedef {{
+ *   entitlement: !ShowcaseEvent,
+ *   isUserRegistered: (boolean|undefined),
```

Comment:
Please add a comment explaining what isUserRegistered and isFromUserAction means or point to other documentation about logging, if available. Not sure if there are other public docs describing these events.

Reply:
Good call, I added some comments! I couldn't find public docs describing these events